thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/util/StringUtils.java

```diff
@@ -0,0 +1,36 @@
+package com.linkedin.thirdeye.datalayer.util;
+
+import com.google.common.base.Joiner;
+import com.google.common.base.Splitter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+
+public class StringUtils {
+  private static Splitter SEMICOLON_SPLITTER = Splitter.on(";").omitEmptyStrings();
+  private static Splitter EQUALS_SPLITTER = Splitter.on("=").omitEmptyStrings();
+
+  private static Joiner SEMICOLON = Joiner.on(";");
+  private static Joiner EQUALS = Joiner.on("=");
+
+  // the following two functions encode and decode properties are moved from class
+  // com.linkedin.thirdeye.controller.mp.function.ThirdEyeAnomalyFunctionUtil
```

Comment:
Remove the class name. Moreover, give us an example of this method's input and output.